src/pudl/analysis/classify_plants_ferc1.py

```diff
@@ -732,6 +735,7 @@ def fuel_by_plant_ferc1(
     ).reset_index()
 
     # Label each plant-year record by primary fuel:
+    df.loc[:, ["primary_fuel_by_cost", "primary_fuel_by_mmbtu"]] = pd.NA
```

Comment:
So here you are overwriting the string "nan" values with real NA values? But we still don't know what changed to introduce them here?  Oh wait didn't @jdangerx do something with null or empty strings / NA values in the XBRL extractor recently? But in your screenshot there are "nan" values in both 1994 and 2022 data.

Reply:
these two columns don't exist before the for/try/except loop. they are being introduced in this loopy masky setup.

WHICH SEEMS NORMAL/FINE. but for some reason the records which are not getting labeled w/ a fuel (i.e. the  highest cost/mmbtu fuel does not meet the 50% threshold) at the end of this loopy loopy had `"nan"` primary fuels.

Reply:
Okay thanks for showing me your debugging. I'm convinced this is a new pandas behavior somehow and now an upstream data issue. Seems very strange though.